website/admin.py

```diff
@@ -1,10 +1,37 @@
+from datetime import date
+
 from flask_babel import gettext
 
 from website.auth import requires_login, current_user, is_admin, pick
 import utils
-from flask import request, g
+from flask import request
 from flask_helpers import render_template
-import hedyweb
+
+
+def get_class_stats_last_week(students):
```

Comment:
The get_class_stats_last_week and get_class_stats_total are identical except for the argument passed to the get_program_stats on the first line. Perhaps we could reduce the repetition.
Additionally, I think this logic belongs to the statistics.py module as it already contains similar logic. Probably we could even reuse (or adapt) some of the transformation functions there?

Reply:
Fixed! Code is moved to statistics.py and the two functions are merged to remove duplicate code.